### PR TITLE
Add card list page

### PR DESCRIPTION
**Pull request by agubler**
work done by @Tomdye

### CODE REVIEW

File: monster-cards/src/widgets/card/createCardSummary.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	score: number;
+	id: string;
+	cardId: string;
+}
+
+type CardSummaryOptions = RenderMixinOptions<CardSummaryState>;
+
+export type CardSummary = RenderMixin<CardSummaryState> & StatefulChildren<Child>;
+
+const create = createRenderMixin
```
Comment (agubler) [inline monster-cards/src/widgets/card/createCardSummary.ts]: Let's name this better

Comment (Tomdye) [inline monster-cards/src/widgets/card/createCardSummary.ts]: I'd been naming them all `create`. do you feel there is a need to rename the `create` function inside `createCardSummary` to `createCardSummary` ?

Comment (agubler) [inline monster-cards/src/widgets/card/createCardSummary.ts]: Seems that is the pattern to give a more meaningful name, although as the default export you can it what ever you want on import anyway. The preexisting factories all had more explicit names.

**Comment by Tomdye** [comment]
PR comments addressed

**Pull request merged**